services/management/dbaas/kubernetes_server.go

```diff
@@ -66,6 +70,24 @@ func (k *kubernetesServer) Enabled() bool {
 	return settings.DBaaS.Enabled
 }
 
+// getOperatorStatus exists mainly to assign appropriate status when installed operator is unsupported.
+// dbaas-controller does not have a clue what's supported, so we have to do it here.
+func (k kubernetesServer) getOperatorStatus(ctx context.Context, operatorType string, operatorVersion string) (dbaasv1beta1.OperatorsStatus, error) {
```

Comment:
```suggestion
func (k kubernetesServer) convertOperatorStatus(ctx context.Context, operatorType string, operatorVersion string) (dbaasv1beta1.OperatorsStatus, error) {
```

Reply:
I think `convert` implies some other status we convert from. But the input is not status.

Reply:
Agree.